### ISSUE TITLE
Proposal: Move this project to the @rust-mobile organization?

### ISSUE THREAD

**Issue by madsmtm** [BrainiumLLC/cargo-mobile#128]
Hi there @BraniumLLC.

I'm the current macOS/iOS maintainer of [Winit](https://github.com/rust-windowing/winit), and am looking to improve the user story for developing Rust applications for iOS.

In the past few years, the ecosystem for making and running mobile bundles has been kinda fractured, with Tauri forking it to create [`cargo-mobile2`](https://github.com/tauri-apps/cargo-mobile2), and solutions like [`xbuild`](https://github.com/rust-mobile/xbuild), [`cargo-dinghy`](https://github.com/sonos/dinghy), [`crossbow`](https://github.com/dodorare/crossbow), [`tugger`](https://gregoryszorc.com/docs/pyoxidizer/main/tugger.html) and many others popping up, all filling the same niche as `cargo-mobile`. I suspect part of the reason is that this project is effectively unmaintained.

I'd like to rectify that, so to that end **I propose moving this repository to the @rust-mobile organization**.

This organization originally sprang out of @rust-windowing around a year ago, when there was a need to consolidate the Android crates, and so far it has worked quite well!

My hope is that such a move will enable more maintenance (I would be doing it for `cargo-mobile` to begin with) and code-sharing between these projects (most of the aforementioned tools have ended up re-implementing a lot of the same logic to make application bundles and launching simulators), as well as paving the path for having a much clearer user story for Rust mobile development.

---

I recognize that this is quite a big ask that could potentially be seen as a hostile takeover, [which have been seen in open source before](https://en.wikipedia.org/wiki/UBlock_Origin), I'm not really sure how to alleviate those concerns, other than openly talking about them? In any case, please treat this proposal as just that, a proposal, you have every right to refuse it!

Lastly, I wanna take this opportunity to thank you for the amazing work you and your company has provided to the Rust community in the past, you're all great! ❤️

Tagging people that I know are or were [members of the organization](https://github.com/orgs/BrainiumLLC/people), for visibility: @Ensath, @spensbot, @francesca64, @ArthurKValladares, @rsnively, @logan-brainium, @mtak-.

<sub>Feel free to state if you think I've been too harsh or rude in this issue, it's always difficult to talk about these sorts of things in open source.</sub>

**Comment by rsnively** [comment]
Hey @madsmtm, I'm one of the people still over at Brainium. Thank you for your interest in revitalizing this project. It would be great to see it become the open source solution it was designed to be (we switched to a private repo a while ago as our business needs diverged from what made sense to share publicly). I will bring it up at work this week and get back to you. Cheers!

**Comment by zeerooth** [comment]
[tauri-apps/cargo-mobile2](https://github.com/tauri-apps/cargo-mobile2) is very well maintained and there have been made quite a lot of progress with adding new features, templates and fixing bugs since abandonment of cargo-mobile by Brainium 2 years ago. Therefore if anything I think that that fork should be the mainstream one. I actually emailed some people at Brainium some time ago about archiving `cargo-mobile` and pointing people to Tauri's `cargo-mobile2` in the README but I got not response.

**Comment by madsmtm** [comment]
> [tauri-apps/cargo-mobile2](https://github.com/tauri-apps/cargo-mobile2) is very well maintained and there have been made quite a lot of progress

Yeah, I'm well aware, my next step in this process would be to propose the same move for Tauri's fork, and then I'd merge the issue trackers and crates.io crates, so that all the open issues and GitHub history isn't lost.

But one thing at a time ;)

**Comment by madsmtm** [comment]
Hey @rsnively, do you have an update on this? Totally cool that it takes some time, just want to know if something is happening
